Dockerfile

```
@@ -9,9 +9,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         python3-setuptools \
         # Required for venv setup
         python3-venv \
-        # Required to build RLE module and dbus-python (GUI)
+        # Required to build RLE module (GUI)
```

```suggestion
        # Required to build RLE module
```
The `GUI` mention was referring to dbus-python (a GUI related dependency). Needs to be removed now.